Provis.Core/Services/WorkspaceService.cs

```diff
@@ -97,7 +97,7 @@ public async Task SendInviteAsync(InviteUserDTO inviteDTO, string ownerId)
             else
             {
 
-                if (checkRole.RoleId == WorkSpaceRoles.MemberId || checkRole.RoleId == WorkSpaceRoles.ViewerId)
+                if (checkRole.RoleId == (int)WorkSpaceRoles.MemberId || checkRole.RoleId == (int)WorkSpaceRoles.ViewerId)
```

Comment:
Let's make check by convert to enum

Reply:
this check was a temporary solution to check if the user is owner or maneger, now for it is policy, so i removed this check